src/ubuntu/build-scripts/build-rootfs.sh

```diff
@@ -12,68 +12,66 @@ lldb=${4:-}
 
 dockerCrossDepsTag="${DOCKER_REPO:-mcr.microsoft.com/dotnet-buildtools/prereqs}:${os}-crossdeps"
 
-# If argument three was set, use that as the only arch, otherwise use default list of arches : 'arm'
-crossArchArray=(${archArg:-'arm'})
```

Comment:
The Bash syntax isn't the same as Bourne shell so this line was failing to the change.  But an array isn't really necessary since only one element is ever used.  I've simplified the script to no longer enumerate an array.

Reply:
Thanks for the simplification/refactor.  This is something that has changed over time (support for the archArg was added after the initial implementation).